README.md

```diff
@@ -198,6 +155,47 @@ To run all the tests use the rake task, which sets some environment variables fo
 Deployment is through capistrano. To deploy a branch other than "main", prepend an environment variable to your deploy command, e.g.:
 `BRANCH=my_feature bundle exec cap staging deploy`
 
+## When a location changes in Alma:
+
+#### 1. Update the local dev environment:
+
+Load the rails console
+1.`bundle exec rails c`
+
+Delete the existing data from the location tables and repopulate them by pulling data from Alma.
+2.`LocationDataService.delete_existing_and_repopulate`
+
+Generate the following files: marc_to_solr/translation_maps/location_display.rb and marc_to_solr/translation_maps/locations.rb
+3.`LocationMapsGeneratorService.generate`
+
+Copy the content from the generated .rb files into the .tmpl.rb files
+4.`cp marc_to_solr/translation_maps/location_display.rb marc_to_solr/translation_maps/location_display.rb.tmpl`
+`cp marc_to_solr/translation_maps/locations.rb marc_to_solr/translation_maps/locations.rb.tmpl`
+
+Load locally the rails server and make sure that the locations have been updated.
```

Comment:
perhaps note that we can do this by going to localhost:3000/locations/holding_locations

Reply:
yes I will add this @hectorcorrea, thank you.